MCprep_addon/tracking.py

```diff
@@ -386,35 +407,68 @@ def set_tracker_json(self):
 		if self._tracker_json is None:
 			raise ValueError("tracker_json is not defined")
 
+		# Placeholder struc before saving to class
+		_json = {
+			"install_date": None,
+			"install_id": None,
+			"enable_tracking": False,
+			"status": None,
+			"metadata": {}
+		}
+
+		valid_tracker = False
 		if os.path.isfile(self._tracker_json) is True:
 			with open(self._tracker_json) as data_file:
-				self.json = json.load(data_file)
+				jdata = json.load(data_file)
+
+			_json = jdata
+			if self._verbose:
+				print(f"{self._addon}: Read in json settings from tracker file")
+			if jdata.get("install_id") in SKIP_IDS:
```

Comment:
I would add a comment explaining what happened (with a link to #491) that lead to this check. I'd also say that `SKIP_IDS` should be renamed to `BAD_IDS` or something along those lines.

Reply:
Good call to add the git issue link. I think doing so at the top makes more sense. On the rename, I decided to go with INVALID_IDS since it's a bit more telling how we treat them, and something could be invalid for any number of reasons (whereas 'bad' in my mind is somehow more negative implication, albeit the most likely case)